actions/ui-check/tests/e2e/specs/page/index.test.js

```diff
@@ -13,21 +14,13 @@ import {
 	getElementPropertyAsync,
 } from '../../utils';
 
-// TODO: either dynamically fetch a list of URLs to check (REST API or site maps?)
-// or import the theme test content dataset and hard-code a list of URLs based on that.
-let urls = [
-	[
-		'/', // URL
-		'', // Query string starting with '?'
-		'home', // Body class to expect
-	],
-	[ '/', '?p=1', 'postid-1' ],
-	[ '/', '?page_id=2', 'page-id-2' ],
-	[ '/', '?author=1', 'author-1' ],
-	[ '/', '?cat=1', 'category-1' ],
-	[ '/', '?feed=rss2', '' ], // Feeds should probably be handled by separate tests.
-	// ...more pages
-];
+import site_info from '../../../../../../config/siteinfo.json';
```

Comment:
Yowza. Since this config technically doesn't need to get mounted, we could probably just add the `package.json` script that is currently in the root folder in the `actions/ui-check/package.json`?

Reply:
> Yowza. Since this config technically doesn't need to get mounted, we could probably just add the `package.json` script that is currently in the root folder in the `actions/ui-check/package.json`?

You mean put the `install:siteinfo` script into the ui-check package.json? Sure. What about the location of the `siteinfo.json` file? I put it in the config dir to mirror `download:accessibility`, but I guess placing it in the same dir as `index.test.js` would mirror the way the a11y tests use `pages.json`.

Reply:
At this moment in time, I don't see how it will be used outside the UI tests. So maybe just move it down for simplicity.

Reply:
Done. The other thing I'm not certain about is using `localhost:8889` in the `package.json` like that. Is there an environment var or something I can use instead?

Reply:
https://www.npmjs.com/package/@wordpress/env

Try: WP_ENV_TESTS_PORT